src/pages/components/datepicker.mdx

```diff
@@ -0,0 +1,117 @@
+  </Use>
+</Usage>
+
+## How to use it
+
+### Default
+
+The typical usage of a Datepicker component.
+
+import Default from '../../examples/datepicker/DatepickerDefault.tsx';
+import DefaultCode from '!!raw-loader!../../examples/datepicker/DatepickerDefault.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Custom date format
+
+Dates are formatted using the
+[Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat)
+utility. For other formats, consider libraries like
+[momentjs](https://momentjs.com/) or [date-fns](https://date-fns.org/).
+
+import Custom from '../../examples/datepicker/DatepickerCustom.tsx';
+import CustomCode from '!!raw-loader!../../examples/datepicker/DatepickerCustom.tsx';
+
+<CodeExample code={CustomCode}>
+  <Custom />
+</CodeExample>
+
+### Datepicker range
+
+Datepicker range works in combination with two text inputs. It displays a
+calendar that lets users choose a start and end date.
+
+import Range from '../../examples/datepicker/DatepickerRange.tsx';
+import RangeCode from '!!raw-loader!../../examples/datepicker/DatepickerRange.tsx';
+
+<CodeExample code={RangeCode}>
+  <Range />
+</CodeExample>
+
+### Localization
+
+Dates are localized using the
+[Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat)
+utility. For other formats, use `customParseDate` prop, and formatting libraries
+like [momentjs](https://momentjs.com/) or [date-fns](https://date-fns.org/).
+Customize language, region, and calendar type with a [BCP 47 language
+tag](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat#Parameters).
+
+import Localization from '../../examples/datepicker/DatepickerLocalization.tsx';
+import LocalizationCode from '!!raw-loader!../../examples/datepicker/DatepickerLocalization.tsx';
+
+<CodeExample code={LocalizationCode}>
+  <Localization />
+</CodeExample>
+
+### Size
+
+Datepicker can be default or compact.
+
+import Size from '../../examples/datepicker/DatepickerSize.tsx';
+import SizeCode from '!!raw-loader!../../examples/datepicker/DatepickerSize.tsx';
+
+<CodeExample code={SizeCode}>
+  <Size />
+</CodeExample>
+
+### Time window
+
+Options before and after a certain date can be disabled, limiting users to a specific time window.
+
+import TimeWindow from '../../examples/datepicker/DatepickerTimeWindow.tsx';
+import TimeWindowCode from '!!raw-loader!../../examples/datepicker/DatepickerTimeWindow.tsx';
+
+<CodeExample code={TimeWindowCode}>
+  <TimeWindow />
+</CodeExample>
+
+## Configuration
+
+<Configuration reactPackage={props.data.mdx.package} components={props.data.mdx.components} />
+
+## API
+
+### Datepicker
+
+<PropSheet components={props.data.mdx.components} componentName="datepicker" />
+
+### DatepickerRange
+
+<PropSheet components={props.data.mdx.components} componentName="datepickerrange" />
+
```

Comment:
Need H4 placeholders for `DatepickerRange.Calendar`, `DatepickerRange.End`, and `DatepickerRange.Start`